docs/source/usage/validation/index.rst

```diff
@@ -17,47 +17,11 @@ starfish validate --help will provide instructions on how to use the tool:
 Examples
 --------
 
-As a simple example, we can show that the :ref:`synthetic experiment <cli_build>`
-created in the previous section is valid:
-
 ::
 
     starfish validate experiment tmp/experiment.json > /dev/null && echo ok
 
 
-Building a :ref:`synthetic experiment <cli_build>` can provide you with a template that
-you can use to model your own data. If you then modify that experiment incorrectly, you
-might see the following validation warning:
-
-::
-
-    $ starfish validate experiment tmp/experiment.json
```

Comment:
Do we want to preserve some documentation about `starfish validate`?

Reply:
I still have just the validate documentation in but I took out the builder bit

Reply:
It is plausible that we want to retain the example about validating an existing experiment.